index.bs

```diff
@@ -9256,6 +9372,22 @@ for their contributions as our W3C Team Contacts.
     "date": "June 2019"
   },
 
+  "RFC9052": {
+    "authors": ["Jim Schaad"],
+    "title": "CBOR Object Signing and Encryption (COSE): Structures and Process",
+    "href": "https://datatracker.ietf.org/doc/rfc9052/",
+    "status": "IETF Internet Standard",
+    "date": "August 2022"
+  },
+
+  "RFC9053": {
+    "authors": ["Jim Schaad"],
+    "title": "CBOR Object Signing and Encryption (COSE): Initial Algorithms",
+    "href": "https://datatracker.ietf.org/doc/rfc9053/",
+    "status": "RFC Informational",
+    "date": "August 2022"
+  },
+
```

Comment:
These are already in the Bikeshed index, and aren't referenced in this extension anyway. Please revert this change.

Reply:
Done.